src/containers/FiveDay.js

```diff
@@ -1,20 +1,19 @@
 import React, { Component } from 'react';
-import { Link } from 'react-router';
+// import { Link } from 'react-router';
 import currentWeather from '../data/fake-current';
 
 export default class FiveDay extends Component {
 
   render() {
     return (
-      <div id="fiveDayForecast">
-        <p id="forecast">
-          <h1>Temp: {currentWeather.main.temp}</h1>
-          <h1>Humidity: {currentWeather.main.humidity}</h1>
-          <h1>Pressure: {currentWeather.main.pressure}</h1>
-          <h1>Min Temp: {currentWeather.main.temp_min}</h1>
-          <h1>Max Temp: {currentWeather.main.temp_max}</h1>
-        </p>
-
+      <div className="fiveDayForecast">
+        <ul className="forecast"><h2>Extended Forecast for Omaha</h2>
+          <li>Temp: {currentWeather.main.temp}</li>
```

Comment:
It might make sense to just do something like this:

```js
const { temp, humidity, pressure, temp_min: tempMin, temp_max: tempMax } = currentWeather.main;
```

That way you can save yourself the hassle of typing `current.main` all the time.